.github/workflows/unittests.yml

```
@@ -0,0 +1,30 @@
+name: unittests
+on:
+  - push
```

To avoid double CI for branches on `tommoral` remote:
```suggestion
  - push:
     branches:
        - 'main'
```